loklak.php

```diff
@@ -136,4 +136,17 @@ public function user($name, $followers=null, $following=null) {
 			return json_encode($request, true);
 		}
 	}
+
+	public function settings() {
+		$this->requestURL = $this->baseUrl . '/api/settings.json';
```

Comment:
Since `settings.json` is a private API and restricted to localhost only, It's best to write it as 
`$this->requestURL = 'http://localhost:9000/api/settings.json'`